src/js/components/List/List.js

```diff
@@ -409,7 +471,7 @@ const List = React.forwardRef(
                 }
 
                 let clickProps;
-                if (onClickItem && !onOrder) {
+                if (onClickItem && !onOrder && !isPinned) {
```

Comment:
Why not allow clicking on a pinned item? That seems like the domain of disabled.

Reply:
Great point, I'll make that change.